node-js-basics-101/node_modules/accepts/index.js

```diff
@@ -0,0 +1,231 @@
+/*!
+ * accepts
+ * Copyright(c) 2014 Jonathan Ong
+ * Copyright(c) 2015 Douglas Christopher Wilson
+ * MIT Licensed
+ */
+
+'use strict'
+
+/**
+ * Module dependencies.
+ * @private
+ */
+
+var Negotiator = require('negotiator')
+var mime = require('mime-types')
```

Comment:
__[jshint]__
* 'require' is not defined.

<!-- Comment by [SideCI](https://sideci.com) -->